spec/validation/meta_yml/unknown_keys_spec.rb

```diff
@@ -14,12 +14,10 @@ module HathiTrust
         before(:each) { allow(mocked_sip).to receive(:meta_yml) .and_return(valid_yaml) }
 
         it_behaves_like "a validation with a valid package"
-
-        it "does not return any messages" do
-          expect(validation.validate.length).to be(0)
-        end
+        it_behaves_like "a validation that returns no messages"
```

Comment:
I like having this along with the above.  It's for validations that do not emit warnings either, correct?